website/client/router.js

```diff
@@ -101,6 +103,14 @@ export default new VueRouter({
             },
           ],
         },
+        { name: 'challenges', path: 'challenges', component: Page },
+      ],
+    },
+    {
+      path: '/user',
```

Comment:
this too seems a duplicate, maybe some type of conflict with `develop` but there's already an user entry